grafana-plugin/src/containers/EscalationVariants/EscalationVariants.module.scss

```diff
@@ -21,9 +21,17 @@
   overflow: scroll;
 }
 
-.schedule-table {
+.table {
   height: 120px;
   overflow: auto;
+
+  & tr:hover {
```

Comment:
What's the usage of `&` in these 2 examples?

Reply:
`.table tr:hover` and `.table tbody tr:nth-child(odd)`

Reply:
Right, I get that, but is it needed? I don't see its purpose.